Libraries/LibJS/Interpreter.cpp

```diff
@@ -132,29 +132,31 @@ void Interpreter::exit_scope(const ScopeNode& scope_node)
 
 void Interpreter::set_variable(const FlyString& name, Value value, bool first_assignment)
 {
-    for (auto* environment = current_environment(); environment; environment = environment->parent()) {
-        auto possible_match = environment->get(name);
-        if (possible_match.has_value()) {
-            if (!first_assignment && possible_match.value().declaration_kind == DeclarationKind::Const) {
-                throw_exception<TypeError>("Assignment to constant variable");
+    if (m_call_stack.size())
```

Comment:
Please wrap multi-line consequent blocks in `{` and `}` :)

Reply:
okies